.github/workflows/main.yml

```diff
@@ -11,9 +11,37 @@ jobs:
   macos:
     runs-on: macOS-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
       with:
         fetch-depth: 1
+    - name: Prepare ENV
+      run: |
+        TAG_NAME="7.0.0"
+        if [[ "${{ github.event_name }}" == "release" ]]; then
+          TAG_NAME=$(bash ./dist/osx/build.sh get_new_tag_version)
```

Comment:
I'm not sure what you means "just write the tag name manually in this case" 🤔 

The reason is that I'm now sure how `TAG_NAME=$(./dist/get-tag-name.sh)` work since I couldn't test it because we have to merge into the master in order to get the result.

From what I see, `get-tag-name.sh` just simply get a current tag without creating a new one, but we need to bump a tag to release DEV doesn't we?

Reply:
>I'm not sure what you means "just write the tag name manually in this case" 
I mean just writing `TAG_NAME="7.5.50" or whatever`. Do we need to create an actual tag when we are just testing?

>From what I see, get-tag-name.sh just simply get a current tag without creating a new one, but we need to bump a tag to release DEV doesn't we?
Yes, true. The GitHub Actions is already creating new tags, see `pr_tagger.yml`. You can verify it by disabling the buildbot and committing something to `master`.

Reply:
Ah, I see your point. Updating now 🙌